common/components/controllers/AboutUsController.jsx

```diff
@@ -7,11 +7,13 @@ import type {ProjectDetailsAPIData} from '../utils/ProjectAPIUtils.js';
 import cdn, {Images} from '../utils/cdn.js';
 import Headers from "../common/Headers.jsx";
 import Person from '@material-ui/icons/Person';
-// TODO: Create metrics event for this page, then import metrics and use it
-// import metrics from "../utils/metrics.js";
+import BioModal from "../componentsBySection/AboutUs/BioModal.jsx";
 
 type State = {|
   aboutUs: ?ProjectDetailsAPIData,
+  showBiographyModal: boolean,
+  modalPerson: object,
```

Comment:
If modalPerson accepts multiple types, you can use Flow unions.  https://flow.org/en/docs/types/unions/

Reply:
I read the flow docs you linked on unions and I think I've set it correctly. All the key/value pairs inside the modalPerson object are numbers or strings. I'm not sure though, so if I got that wrong just let me know.